### PR TITLE
fix task details show

### PR DESCRIPTION
**Pull request by tomlovesgithub**
This ticket addresses a bug which is preventing applications from showing.
Bug introduced in #1314 

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_

### CODE REVIEW

File: src/views/Exercise/Applications/Application.vue [hunk 1]
```diff
@@ -1463,44 +1466,80 @@
...
+                  v-if="item.taskDetails.totalDaysInRole"
+                  class="govuk-summary-list__row"
+                >
+                  <dt class="govuk-summary-list__key">
+                    Total number of days engaged in this role
+                  </dt>
+                  <dd class="govuk-summary-list__value">
+                    {{ item.taskDetails.totalDaysInRole }}
+                  </dd>
+                </div>
```
Comment (mbrookeswebdev) [inline src/views/Exercise/Applications/Application.vue:1541]: @tomlovesgithub should we have a v-else 'No answer provided' here as well (for consistency)?

**Review by mbrookeswebdev — COMMENTED** [review]
👍

**Review by mbrookeswebdev — APPROVED** [review]

**Review by joy-ade — APPROVED** [review]
Approved by product team

**Review by warrensearle — APPROVED** [review]
👍

**Review by HalcyonJAC — APPROVED** [review]
👍

**Pull request merged**